src/mixins/menubar.js

```diff
@@ -115,6 +127,7 @@ export default [
 	},
 	{
 		label: t('text', 'Unordered list'),
+		keyboard: t('text', 'Shift') + '+' + t('text', 'Ctrl') + '+8',
```

Comment:
I feel like constructing these strings could be moved to a helper function. Something like:
```
keyboardShortcut('8', shift: true, ctrl: true)
```

Reply:
Did this now, thanks to pair programming with @azul  :)